### PR TITLE
providing new feature: Heat index

### PR DESCRIPTION
**Pull request by t-m-z**
for review and discussion; I am not sure if all the additional information is really necessary.

For future releases I could add Humidex Index and Summer Simmer Index:
https://myscope.net/hitzeindex-gefuehle-temperatur/#:~:text=%20Summer-Simmer-Index%20%28SSI%29%20Berechnung%20%201%20SSIC%20%E2%80%93,F%20%E2%80%93%20relative%20Luftfeuchtigkeit%20in%20%25%20More%20

### CODE REVIEW

File: lib/widgets/tools/science_and_technology/heat.dart [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+          GCWOutputText(
+              text: _currentOutput + degree
+          ),
+
+          GCWTextDivider(
+              text: i18n(context, 'heat_hint')
+          ),
+
+          GCWOutputText(
+                text: hint
```
Comment (S-Man42) [inline lib/widgets/tools/science_and_technology/heat.dart:122]: Please have a look at the indents :)

**Review by S-Man42 — COMMENTED** [review]
Looks great! Just add the de.json file :)

File: lib/widgets/registry.dart [hunk 2]
```diff
@@ -411,6 +412,12 @@ class Registry {
         category: ToolCategory.CRYPTOGRAPHY,
         searchStrings: SEARCHSTRING_HASHES
       ),
+      GCWToolWidget(
+        tool: Heat(),
+        i18nPrefix: 'heat',
+        category: ToolCategory.SCIENCE_AND_TECHNOLOGY,
+        searchStrings: 'temperature temperatur humidity luftfeuchtigkeit hitze'
```
Comment (S-Man42) [inline lib/widgets/registry.dart:419]: I'd add as in Windchill: gefuehlte temperatur apparent temperature

File: lib/widgets/tools/science_and_technology/heat.dart (hunk 1 above)
Comment (t-m-z) [reply lib/widgets/tools/science_and_technology/heat.dart:122]: done

File: lib/widgets/registry.dart (hunk 2 above)
Comment (t-m-z) [reply lib/widgets/registry.dart]: done

**Comment by t-m-z** [comment]
> Looks great! Just add the de.json file :)

Done - however, I have no clue why git removed the content

**Pull request merged**